app/src/main/res/values/strings.xml

```diff
@@ -593,6 +593,7 @@
     <string name="app_language_title">App language</string>
     <string name="systems_language">System default</string>
     <string name="dynamic_seek_duration_description">%s seconds</string>
+    <string name="new_seek_duration_toast">Due to ExoPlayer contraints the seek duration was set to</string>
```

Comment:
Shouldn't this have a `%s` indicating where the "X seconds" string should be inserted? For English it is ok for this to be at the end, but for other languages this could be different.

Reply:
Good point, didn't think of other languages having a different arrangement